views/movies/new-movie.hbs

```diff
@@ -0,0 +1,14 @@
+<form action='/movies/create' method='post'>
+<label for="title">title</label>
+<input type="text" title="title" id="title" />
```

Comment:
Be careful here, you passed `title` instead of `name` so when you create a movie it isn't creating a title for it.

`<input type="text" name="title" id="title" />`